jekyll/_cci2/credits.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: classic-docs
+title: Using Credits
+categories: [how-to]
+description: How to leverage CircleCI credits
+---
+
+This document describes the basics of using credits with CircleCI. If you are on the legacy CircleCI container-based plan you may want to consider consulting the document on [using containers]({{site.baseurl}}/2.0/containers). If you want to switch from using containers to using credits, please [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) requesting so.
```

Comment:
That's a fair question. How do we want to proceed regarding how a user is coming to this document? If they are evaluating whether the should switch to the Usage plan, we should definitely change that, possibly also change the title to "CircleCI Usage Plan" or something similar..., but if they are already on a Usage plan, it might be safer to assume understanding of credits. 

Either way, I do agree we should start more concretely with language about the `Usage` plan, then introduce credits, perhaps.